### PR TITLE
HSEARCH-4157 Raise all the timeouts to make all tests working

### PR DESCRIPTION
**Pull request by fax4ever**
https://hibernate.atlassian.net/browse/HSEARCH-4157

### CODE REVIEW

File: integrationtest/mapper/orm-realbackend/src/test/java/org/hibernate/search/integrationtest/mapper/orm/realbackend/routing/RoutingBridgeRoutingKeyIT.java [hunk 1]
```diff
@@ -52,6 +52,10 @@ public void setup() {
 		entityManagerFactory = setupHelper.start()
 				.withProperty( HibernateOrmMapperSettings.AUTOMATIC_INDEXING_SYNCHRONIZATION_STRATEGY,
 						AutomaticIndexingSynchronizationStrategyNames.READ_SYNC )
+				// This call in testLifecycle to Elasticsearch sever may exceed 30 seconds,
+				// from some machine. This is a server (not Hibernate Search) latency.
+				// E.g: > query parameters {} and 1 objects in payload in 30617ms.
+				.withProperty( "hibernate.search.backend.read_timeout", "60000" )
```
Comment (yrodiere) [inline integrationtest/mapper/orm-realbackend/src/test/java/org/hibernate/search/integrationtest/mapper/orm/realbackend/routing/RoutingBridgeRoutingKeyIT.java:58]: Come on, this test takes 5 seconds on my machine. No way this takes more than 30s on yours unless there's yet another problem... Let's talk about it on Zulip.

File: integrationtest/performance/backend/elasticsearch/src/main/java/org/hibernate/search/integrationtest/performance/backend/elasticsearch/testsupport/ElasticsearchBackendHolder.java [hunk 2]
```diff
@@ -44,6 +45,9 @@ protected ConfigurationPropertySource getDefaultBackendProperties(TemporaryFileH
 
 		map.put( ElasticsearchIndexSettings.SCHEMA_MANAGEMENT_MINIMAL_REQUIRED_STATUS, IndexStatus.YELLOW );
 
+		// In some machine (e.g. MacOS machines) there is an extra latency
+		map.put( ElasticsearchBackendSettings.READ_TIMEOUT, "120000" );
```
Comment (yrodiere) [inline integrationtest/performance/backend/elasticsearch/src/main/java/org/hibernate/search/integrationtest/performance/backend/elasticsearch/testsupport/ElasticsearchBackendHolder.java:49]: Let's stop a minute and ask ourselves why a single request would need a 2 minute timeout, shall we? Let's investigate the test that prompted you to add this, on Zulip.

**Review by yrodiere — COMMENTED** [review]
I can live with the JSR-352 tests being slow, but the other changes are concerning. Let's talk on Zulip.

**Review by yrodiere — APPROVED** [review]

**Comment by yrodiere** [comment]
Merged, thanks!

Related discussion for anyone interested: https://hibernate.zulipchat.com/#narrow/stream/132092-hibernate-search-dev/topic/HSEARCH-4157

**Comment by fax4ever** [comment]
thank you!

**Pull request merged**